<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NFOBuilder for Plex</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/plex-nfo.ico</normaloff>:/resources/plex-nfo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="showNameLabel">
      <property name="text">
       <string>Show Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="showNameInput">
      <property name="placeholderText">
       <string>The shows name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="showDateLabel">
      <property name="text">
       <string>Performance Date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="showDateInput">
      <property name="placeholderText">
       <string>YYYY-MM-DD</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="showLocationLabel">
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="showLocationInput">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Location/Studio</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="showTaglineLabel">
      <property name="text">
       <string>Tag Line</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="showTaglineInput">
      <property name="placeholderText">
       <string>A tagline for the show</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="showSynopsisLabel">
      <property name="text">
       <string>Synopsis</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="showSynopsisInput">
      <property name="placeholderText">
       <string>A brief synopsis of the show, or notes</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="showPosterLabel">
      <property name="text">
       <string>Show Poster URL</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="showPosterInput">
      <property name="placeholderText">
       <string>Full URL to image to use as Poster</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="CastLabel">
      <property name="text">
       <string>Cast Members</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QTableWidget" name="castTable">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="rowCount">
       <number>40</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Actor/Actress Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Character Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Headshot URL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CLICK to search images</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsEnabled</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="9" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="10" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="11" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="12" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="13" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="14" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="15" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="16" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="17" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="18" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="19" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="20" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="21" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="22" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="23" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="24" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="25" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="26" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="27" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="28" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="29" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="30" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="31" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="32" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="33" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="34" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="35" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="36" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="37" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="38" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="39" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="outputFolderLabel">
      <property name="text">
       <string>Output folder</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="outputFolderInput">
      <property name="placeholderText">
       <string>e.g C:/Users/Pete/Desktop</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="2">
     <widget class="QPushButton" name="CreateNFO">
      <property name="text">
       <string>Create NFO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
